Assets/Branch/Branch.cs

```diff
@@ -68,7 +68,7 @@ public class Branch : MonoBehaviour {
      * Get the referring parameters from the initial install.
      */
     public static Dictionary<string, object> getFirstReferringParams() {
-        string firstReferringParamsString = _getFirstReferringParams();
+        string firstReferringParamsString = Marshal.PtrToStringAnsi(_getFirstReferringParams());
```

Comment:
I'm not 100% sure about this, but we typically use UTF8 for everything... Is ANSI going to cause problems?